edx_django_utils/cache/utils.py

```diff
@@ -4,6 +4,7 @@
 import hashlib
 import threading
 
+import waffle  # pylint: disable=invalid-django-waffle-import
```

Comment:
I used waffle instead of the Django setting as we're already using the waffle switch in this repo.

https://github.com/openedx/edx-django-utils/blob/f9836a1fdc051f502ede86d13b465ce1f6d8790d/edx_django_utils/monitoring/internal/middleware.py#L19